tracer/src/Datadog.Trace.Trimming/Datadog.Trace.Trimming.nuspec

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Datadog.Trace.Trimming</id>
+    <version>1.0.0</version>
+    <title>Datadog APM</title>
```

Comment:
We should add something to "Datadog APM" here. Maybe something like "Datadog APM Trimmed App Support"?

Reply:
done in [930ae59](https://github.com/DataDog/dd-trace-dotnet/pull/4195/commits/930ae592fc27808e3ab4a83f490d361396b52b60)